### ISSUE TITLE
[BUG] Accessibility - Web App - h1 must match part of the title

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5144]
**Describe the bug**
h1 elements must match part of the title element content.

AInspector states “h1 elements can be accessed by assistive technologies to identify the page content and to orient users within the website. The h1 element may also be used to identify the website.”

[WCAG 1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/quickref/#info-and-relationships)
[WCAG 2.4.6 Headings and Labels](https://www.w3.org/TR/WCAG21/#headings-and-labels)

**To Reproduce**
1. Go to app.illinois.edu in Firefox
2. Run AINspector
3. Go to the Navigation violations

**Expected behavior**
h1 elements must match part of the title element content.

**Actual behavior**
h1 elements are missing

**Screenshots**

![Image](https://github.com/user-attachments/assets/8ea943f9-d1b8-4ab3-a36d-925d4c1d3524)

**Desktop (please complete the following information):**
PC:
•	OS: Windows 11
•	Device: Dell laptop
•	Browser: Chrome 135.0.7049.116/Firefox 138.0.1

**Additional context**
